test-result-summary-client/src/Build/GitNewIssue.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import { params } from '../utils/query';
+import {  Button, Tooltip, Card } from 'antd';
+import TestBreadcrumb from './TestBreadcrumb';
+import { GithubOutlined } from '@ant-design/icons';
+import { getParams } from '../utils/query';
+import renderDuration from './Duration';
+
+import './table.css';
+
+import moment from 'moment';
+const DAY_FORMAT = 'MMM DD YYYY, hh:mm a';
+
+export default class GitNewissue extends Component {
+    state = {
+        urlParams: null,
+        body: '',
+        title: '',
+    };
+
+    async componentDidMount() {
+        await this.updateData();
+    }
+
+    async updateData() {
+        const { testId, testName, duration, buildId, buildName, 
+            buildUrl, machine, buildTimeStamp, javaVersion, testResult } = getParams( this.props.location.search );
+            
+        let firstSeenFailure, firstFailDate, firstFailUrl, firstFailJavaVersion;
+        let failCount = 0;
+        let machinesMap = {};
+        let machines = new Set();
+        const fetchBuild = await fetch( `/api/getHistoryPerTest?testId=${testId}&limit=100`, {
+            method: 'get'
+        } );
+        const response = await fetchBuild.json();
+
+        for (let i = 0; i < response.length; i ++){
+                const testResult= response[i].tests.testResult;
+                const machine = response[i].machine;
+                if(testResult === 'PASSED') {
+                    break;
+                } else if(testResult === 'FAILED') {
+                    machines.add(machine);
+                    firstSeenFailure = response[i];
+                    machinesMap[machine] = machinesMap[machine] ? machinesMap[machine] + 1 : 1;
+                    failCount++;
+                }
+        }
+
+        let failMachineUrlBody = '';
+        machines.forEach( value => {
+            failMachineUrlBody += `Machine type: ${value}, Number: ${machinesMap[value]}\n`;
```

Comment:
We should be more clear in the message. For example:
The test failed on machineXXX 5 times